packages/devtools-reps/src/reps/test-helpers.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I tried that, but then Jest fails since it wants to pick it up as a test suite:

```
 FAIL  src/reps/tests/test-helpers.js
  ● Test suite failed to run

    Your test suite must contain at least one test.
      
      at onResult (node_modules/jest-cli/build/TestRunner.js:110:20)
```

Any idea how I can fix this?